cli/pom.xml

```diff
@@ -115,17 +115,17 @@
               <configuration>
                 <artifacts>
                   <artifact>
-                    <file>${releaseName}</file>
+                    <file>target/${project.artifactId}-linux-x64.tar.gz</file>
```

Comment:
This should fix the current error of the workflow. I'm not sure if entering the filenames statically is the best way, but the last deployment job in the workflow is running on Ubuntu and the classifier is not working.

Reply:
I am happy with this approach for now. We can still improve, if we think this is problematic.